odl/ufunc_ops/ufunc_ops.py

```diff
@@ -380,21 +384,28 @@ def __repr__(self):
 """
 
 RAW_UFUNC_FACTORY_FUNCTIONAL_DOCSTRING = """
-Create functional with domain/range as real numbers:
+Create functional with real numbers as domain/range:
 
 >>> func = odl.ufunc_ops.{name}()
 """
 
 RAW_UFUNC_FACTORY_OPERATOR_DOCSTRING = """
-Create operator that acts pointwise on a `TensorSpace`
+Create operator that acts pointwise on a `TensorSpace`:
 
 >>> space = odl.rn(3)
 >>> op = odl.ufunc_ops.{name}(space)
 """
 
+# Avoid tons of warnings on the console when testing
+npy_err_old = np.geterr()
+np.seterr(all='ignore')
 
 # Create an operator for each ufunc
-for name, nargin, nargout, docstring in UFUNCS:
+for name, nin, nout, docstring in UFUNCS:
+    if nin == 2:
+        # Currently not supported
+        continue
```

Comment:
We had a bunch of dysfunctional ufunc ops created here.

Reply:
Good catch